monitoring_101.rst

```diff
@@ -1,14 +1,77 @@
 Monitoring, Notifications, and Metrics 101
 ******************************************
+An Operations team is only as responsive as the telemetry data they have.
+If you have very little or no telemetry data, you will only be reactive and
+when an issue is identified you will be slow to respond and will be
+troubleshooting based on hunches and gut feelings because you are in the dark
+about how the system is operating. If, however, you are proactive in adding
```

Comment:
Grammatical nit pick: A run-on sentence, little hard to parse. Can we break that up into multiple sentences?